src/Lean/Meta/Tactic/ElimInfo.lean

```diff
@@ -185,6 +186,7 @@ builtin_initialize
     descr := "custom `casesOn`-like eliminator for the `cases` tactic"
     add   := fun declName _ attrKind => do
       discard <| addCustomEliminator declName attrKind (induction := false) |>.run {} {}
+    delab := fun _ => pure ()
```

Comment:
Perhaps for these noop implementations we could have a comment indicating whether this is actually the desired behaviour, or temporary.

Reply:
I think in almost no case is it "desired behavior", but they differ a lot in the feasibility of adding them. I can explain the reasons for each one if you want. How temporary they are depends on how much we are willing to change the implementation of the individual attribute's state to make it possible to respond to the query, or how willing we are to eat the cost of things like reverse lookup on a discrimination tree.